cbw_api_toolbox/cbw_auth.py

```diff
@@ -0,0 +1,66 @@
+import base64
+import datetime
+import hmac
+from hashlib import sha256, md5
+
+from requests.auth import AuthBase
+
+from cbw_api_toolbox import JSON_CONTENT_TYPE, SIGNATURE_HTTP_HEADER, TIMESTAMP_HTTP_HEADER, CONTENT_HASH_HEADER, \
+    CONTENT_TYPE_HEADER, SIGNATURE_HEADER
+
```

Comment:
Remove this line (double space)

Reply:
voir https://www.python.org/dev/peps/pep-0008/#blank-lines